<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <node pkg="gnss_poser" name="gnss2tfpose" type="gnss2tfpose" output="screen" >
    <param name="input_msg" type="str" value="fix" />
    <param name="output_msg" type="str" value="utm" />
    <param name="buff_epoch" type="int" value="1" />
    <remap from="/gnss_pose" to="fix2mgrs/gnss_pose" />
  </node>

</launch>
